internal/charmstore/store.go

```diff
@@ -549,7 +549,9 @@ func (s *Store) AddCharm(c charm.Charm, p AddParams) (err error) {
 	denormalizeEntity(entity)
 
 	// Check that we're not going to create a charm that duplicates
-	// the name of a bundle. This is racy, but it's the best we can do.
+	// the name of a bundle. This is racy, but it's the best we can
```

Comment:
I'm going to leave this as it was the comment that was already there, just reformatted.